### PR TITLE
Change REQ-4-2 to require user to configure *new* MG activated at fork

### PR DESCRIPTION
**Pull request by ftrader**

### CODE REVIEW

**Comment by deadalnix** [comment]
Good.

File: BUIP-UAHF/buip-uahf-technical-spec.md [hunk 1]
```diff
@@ -94,20 +94,26 @@ NOTE 2: It has been suggested to replace this requirement with a startup
...
-mining on the fork to a minimum of 8MB regardless of user's MG
-configuration.
-
-NOTE 1: It has been suggested that this requirement is miner policy
-and not needed in this BUIP.
+### REQ-4-2 (require user to specify suitable *new* MG at startup)
+
+If BUIP-UAHF is not disabled (see REQ-DISABLE), the client shall require
+the user to specify a new mining generation limit (MG) greater than
+1,000,000 bytes but not exceeding 8,000,000 bytes.
```
Comment (dgenr8) [inline BUIP-UAHF/buip-uahf-technical-spec.md]: <strike>&nbsp;but not exceeding 8,000,000 bytes</strike>

Comment (ftrader) [inline BUIP-UAHF/buip-uahf-technical-spec.md]: Allowing > 8MB could cause the client to produce an invalid fork block....

**Comment by gandrewstone** [comment]
I'm "for" this PR.  It can be implemented very easily with CTweaks.

**Comment by ftrader** [comment]
I am also for this PR. Even if MG is limited to a bump to 8MB at the fork, it can be adjusted by miners using RPC after the fork has activated.

(I express this explicitly since I raised some of these PRs from discussion - it does not automatically mean I support them :-) )

**Comment by sickpig** [comment]
I support this PR. 
Also the amend made by @dgenr8 conditionally to the rejection of #31

**Comment by ftrader** [comment]
@sickpig : As per Slack meeting actions, I have updated 4-2 .

I added a "fork MG" analogous to "fork EB" to make the terminology consistent.

Rationale was also updated, and a note removed because it was redundant.

**Pull request merged**